backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/beaconv2/endpoints/analyses/AnalysesResultSetsItem.java

```diff
@@ -0,0 +1,53 @@
+package org.molgenis.emx2.beaconv2.endpoints.analyses;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class AnalysesResultSetsItem {
+
+  private String id;
+  private String runId;
+  private String biosampleId;
+  private String individualId;
+  private String analysisDate; // todo LocalDate using jackson-datatype-jsr310 ?
+  private String pipelineName;
+  private String pipelineRef;
+  private String aligner;
+  private String variantCaller;
```

Comment:
Checks complain about these variables being unused, but they are used in the API of course. Perhaps later add getters and tests that use all variable getters in order to touch all variables and they are marked as used?

Reply:
Yeah, or you can do a sonar ignore. Something like @java.lang.SuppressWarnings("java:...."). see https://www.baeldung.com/sonar-exclude-violations . You apparantly then have to name the warning you would like to surpress (PR of Tommy has examples, see https://github.com/molgenis/molgenis-emx2/pull/1378

Reply:
Should look into this later, would help to track down any real problems